core/src/main/java/de/gurkenlabs/litiengine/GameRandom.java

```diff
@@ -776,19 +788,25 @@ public String nextAlphabetic(final int length, final boolean lowerCase) {
    * @param originalColor
    *          The original color to be modified.
    * @param colorVariance
-   *          The float value between 0 and 1 defining how strong the new Color's RGB values will deviate from the original Color.
+   *          The float value between 0 and 1 defining how strong the new Color's
+   *          RGB values will deviate from the original Color.
    * @param alphaVariance
-   *          The float value between 0 and 1 defining how strong the new Color's Alpha will deviate from the original Color.
+   *          The float value between 0 and 1 defining how strong the new Color's
+   *          Alpha will deviate from the original Color.
    * @return A pseudo-randomized variant of the original Color.
    */
   public Color nextColor(Color originalColor, float colorVariance, float alphaVariance) {
     if (originalColor == null) {
       return null;
     }
-    int red = MathUtilities.clamp((int) (originalColor.getRed() + (originalColor.getRed() * this.nextFloat(colorVariance) * this.nextSign())), 0, 255);
-    int green = MathUtilities.clamp((int) (originalColor.getGreen() + (originalColor.getGreen() * this.nextFloat(colorVariance) * this.nextSign())), 0, 255);
-    int blue = MathUtilities.clamp((int) (originalColor.getBlue() + (originalColor.getBlue() * this.nextFloat(colorVariance) * this.nextSign())), 0, 255);
-    int alpha = MathUtilities.clamp((int) (originalColor.getAlpha() + (originalColor.getAlpha() * this.nextFloat(alphaVariance) * this.nextSign())), 0, 255);
+    int red = MathUtilities.clamp((int) (originalColor.getRed() + (originalColor.getRed() * this.nextFloat(colorVariance) * this.nextSign())), 0,
+        255);
+    int green = MathUtilities.clamp((int) (originalColor.getGreen() + (originalColor.getGreen() * this.nextFloat(colorVariance) * this.nextSign())),
+        0, 255);
+    int blue = MathUtilities.clamp((int) (originalColor.getBlue() + (originalColor.getBlue() * this.nextFloat(colorVariance) * this.nextSign())), 0,
+        255);
+    int alpha = MathUtilities.clamp((int) (originalColor.getAlpha() + (originalColor.getAlpha() * this.nextFloat(alphaVariance) * this.nextSign())),
+        0, 255);
```

Comment:
I'd add a spotless off marker for this section, or increase the line limit by like 20 characters.